.github/workflows/modem_sim__build.yml

```
@@ -11,17 +11,21 @@ jobs:
   build_modem_sim:
     if: contains(github.event.pull_request.labels.*.name, 'modem_sim') || github.event_name == 'push'
     name: Build
-    strategy:
-      matrix:
-        idf_ver: ["release-v5.4"]
-    runs-on: ubuntu-22.04
-    container: espressif/idf:${{ matrix.idf_ver }}
+    runs-on: ubuntu-latest
     steps:
       - name: Checkout esp-protocols
         uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)